server/src/controllers/Events.js

```diff
@@ -147,7 +147,7 @@ module.exports = {
     try {
       const eventId = req.body.id;
       const userEmail = req.body.email;
-      const event = await Event.findById(eventId);
+      const event = await Event.findById(eventId).select({"eventName":1}).lean();
```

Comment:
The immediate next line requires `event.date`, that would not be available here.

Reply:
Will fix that